backend/src/main/scala/cromwell/backend/BackendLifecycleActorFactory.scala

```diff
@@ -24,6 +30,19 @@ trait BackendLifecycleActorFactory {
                              initializationData: Option[BackendInitializationData],
                              serviceRegistryActor: ActorRef): Props
 
+  /**
+    * The default implementation of this method ignores the fact that there has been a cache hit and creates props for
+    * a job execution actor.  If a backend has implemented call caching this will not be the desired behavior and this
+    * method should be overridden.
+    */
+  def cacheHitCopyingActorProps(jobDescriptor: BackendJobDescriptor,
+                                initializationData: Option[BackendInitializationData],
+                                serviceRegistryActor: ActorRef,
+                                cachedJobOutputs: JobOutputs): Props = {
+    log.warn("{}: no cache hit copying actor defined, falling back to running the job.", jobDescriptor.key.tag)
+    jobExecutionActorProps(jobDescriptor, initializationData, serviceRegistryActor)
```

Comment:
This version of the code has the factory driving the choice of cache hit copying actor vs job execution actor, but @cjllanwarne correctly pointed out that this choice should be under EJEA's control so EJEA can put itself in the appropriate state.  So rather than return an ActorRef which might be a copying actor or an execution actor, the factory will only return `Option(copyingActor)` or `None`.